app/models/person.rb

```diff
@@ -65,6 +65,10 @@ def max_reward_points
     protocol_subscriptions.map(&:max_reward_points).reduce(0, :+)
   end
 
+  def max_still_earnable_reward_points
+    protocol_subscriptions.active.map(&:max_still_earnable_reward_points).reduce(0, :+)
```

Comment:
`[{a: 1}, {a: 2}, {a: 3}].map{|x| x[:a]}.reduce(0, :+)`

is het zelfde als
`[{a: 1}, {a: 2}, {a: 3}].reduce(0) {|x, y| x+= y[:a]}`

of als
`[{a: 1}, {a: 2}, {a: 3}].sum{|y| y[:a]}`

Reply:
hip. gefixt